apps/site/layouts/layouts.module.css

```diff
@@ -229,8 +229,7 @@
     main {
       @apply max-w-[660px]
         gap-4
-        break-all
-        lg:break-normal;
+        overflow-wrap: anywhere;
```

Comment:
Build is failing. Are you sure this is a valid Tailwind class name? Otherwise you need to wrap it with `[]`

Reply:
I was just researching that - now realizing the original PR is mixing CSS and TW 👎